cmake/cmake-utilities.cmake

```diff
@@ -9,8 +9,6 @@
 #
 function(make_executable name srcs libs)
     add_executable(${name} ${srcs})
-    target_include_directories(${name} PRIVATE ${libcsound_private_include_dirs})
-    target_include_directories(${name} PUBLIC ${libcsound_public_include_dirs})
```

Comment:
moved this line to make_utility. I don't think executables are intended to rely on private headers. My bad from #1796